source/codegen/metadata/nidigital/functions.py

```diff
@@ -2285,6 +2285,28 @@
         ],
         "returns": "ViStatus"
     },
+    
+    "SelectFunction":{
+        "parameters":[
+            {
+                "name":"vi",
+                "direction":"in",
+                "type":"ViSession"
+            },
+            {
+                "name":"channelList",
+                "direction":"in",
+                "type":"ViConstString"
+            },
+            {
+                "name":"function",
+                "direction":"in",
+                "type":"ViInt32",
```

Comment:
Mention in PR description that this API which was using ViInt32 was removed earlier in previous PR #64. But now since our feature branch has support for ViInt32, we are adding back this API. Are there any more APIs using ViInt32 which can be added right now?

Reply:
this API got missed when copying the metadata from base to functions.py. so it was added into the PR